### PR TITLE
Getting_started es-translation

### PR DESCRIPTION
**Pull request by lh1008**
Hey @erciccione 

Here's the es_getting_started translation. Pinging @BlackLotus64 to see if he can check this out. Will try to search for a reviewer, if you know about someone please let him know about this PR. 

Thanks.

### CODE REVIEW

**Comment by BlackLotus64** [comment]
Working from home currently, and yes, absolutely, will have a chance in my afternoon in 8 and half hours.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Empezando a Ayudar a Monero
+
+**_“La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta.”_ - 21 de agosto, 2019**
+---
+
+La comunidad descentralizada de Monero está formada por un grupo de individuos muy dinámico y grupos propagados alrededor del planeta. Un hecho poco conocido y emocionante es que Monero tiene la tercera cuenta de colaboradores más alta en todas las criptomonedas. Adicionalmente, hay muchos grupos de trabajo y proyectos que han evolucionado de la mano de Monero y es relativamente fácil empezar a ayudar.
+
+Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y como PUEDEN involucrarse. Bienvenidos a Monero. Esperamos que se queden.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:8]: _Por favor usen esto como un recurso para aprender sobre Monero, conocer sobre su ecosistema y **cómo** PUEDEN involucrarse._

I would change the "PUEDEN involucrarse" to "PUEDES involucrarte" but is not a grammatical mistake, so it's fine.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+[Desarrolladores](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#developers)
+[Equipos de trabajo](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+[Únete a la conversación e involucrate](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#join)
+[Otras formas de ayudar](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#otherways)
+[Monero 101](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#monero101)
+[Minar Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#mining)
+
+### _Desarrolladores_
+
+Si eres un desarrollador, te invitamos a que te unas a algún equipo.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:19]: _Si eres un desarrollador, te invitamos a que te unas **al equipo**._

Since it just says, "join the team", not "join a team" in the original file.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+La principal manera de empezar, es familiarizarse con la base del código, el estándar del protocolo cryptonote e interactuar con otros desarrolladores. Todos los desarrolladores se comunican a través de IRC en #monero-dev o #monero en los servidores de freenode. Encuentra aquí recursos para que los desarrolladores puedan empezar:
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:21]: It's a suggestion, I believe that _base del código_ it's more known as _código base_. Used Google Translator and it gave the first result only, but the meaning would be the same.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+- Código Fuente - [github.com/monero-project](https://github.com/monero-project)
+- Asuntos(Issues) abiertos - [github.com/monero-project/monero/issues](https://github.com/monero-project/monero/issues)
+- El ecosistema de Monero - [github.com/monero-ecosystem](https://github.com/monero-ecosystem)
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:25]: **Asuntos (Issues)**, there should be a space, shouldn't be?

Another suggestion, **Ecosistema de Monero** would be better, to follow the other bullets translations: is not _**El** laboratorio de investigación de Monero_, and so on.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+- Apuntes de las reuniones de los desarrolladores - [monerobase.com](https://monerobase.com/)
+- Espacios de desarrollo de Monero - [moneroworld.com/development.html](https://moneroworld.com/development.html)
+- Laboratorio de Investigación de Monero - [web.getmonero.org/es/resources/research-lab](https://web.getmonero.org/es/resources/research-lab/)
+
+Los desarrolladores también podrían estar interesados en participar en alguno de los equipos de trabajo existentes dentro de la comunidad de Monero. Pueden encontrar una lista completa de los más activos más abajo.
+
+### _Equipos de Trabajo_
+
+Abajo encontrarás la lista de los equipo de trabajo de Monero que se actualizaran a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no este en la lista, toma la iniciativa y empieza. Existen algunos recursos disponibles para la comunidad de Monero para ayudar al desarrollo efectivo de los equipos de trabajo y sus operaciones. Si tienen preguntas vayan alguno de los foros y pregunten lo que necesiten. Tenemos decenas de colaboradores que pueden ayudarlos a arrancar.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:36]: _Abajo encontrarás la lista de los equipo de trabajo de Monero que se **actualizarán** a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no **esté** en la lista, toma la iniciativa y empieza. _

I would rephrase the next to:

_Si tienes preguntas, ve a alguno de los foros y pregunta lo que necesites. Tenemos decenas de colaboradores que pueden ayudarte a empezar._

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+#### Desarroladores
+Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros software cruciales.
+_¿Dónde empezar?_ [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#developers)
+
+#### Comunidad
+Organizar reuniones, proveer materiales útiles y el Coffee chat mensual.
+_¿Dónde empezar?_ [r/MoneroCommunity](https://www.reddit.com/r/MoneroCommunity/)
+
+#### Localización
+Traducción de materiales relacionados con Monero. Si hablan otro idioma, revisen esto.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:47]: I feel like the translation is using a way to speak to the reader in plural, in example here:

_Si hablan otro idioma, revisen esto._

The original file, as per language (English) uses more like a "you", singular way to refer to the reader, not in plural. It may be interpreted different but a text usually talks to you in singular rather than plural. The main point would be:

_Si hablas otro idioma, revisa esto._

It's open to review since you used that way in other parts, it can be just left as is.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+#### Mastering Monero (Libro - Dominando a Monero)
+Mejorando y actualizando futuras ediciones del libro financiado por la comunidad.
+_¿Dónde empezar?_ [github.com/monerobook/monerobook](https://github.com/monerobook/monerobook)
+
+#### Kit de encuentro de Monero
+Elaboración de herramientas para kits de encuentros de Monero.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/sgp-monero-meetup-kit](https://taiga.getmonero.org/project/sgp-monero-meetup-kit/)
+
+#### Integraciones
+Colaborar para poder integrar a Monero en una variedad de servicios, esto incluye procesadores de pagos, casas de cambio descentralizadas y más.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:110]: Suggestion to improve and reduce phrase:

_Colaborar para poder integrar a Monero en una variedad de servicios, **incluyendo** procesadores de pagos, casas de cambio descentralizadas y más._

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 8]
```diff
@@ -0,0 +1,197 @@
...
+_¿Dónde empezar?_ [taiga.getmonero.org/project/serhack-monero-integrations](https://taiga.getmonero.org/project/serhack-monero-integrations/) & #monero-integrations
+
+#### Labratorio de transmisiones de Monero
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:113]: It is **Laboratorio**, not Labratorio, it's a typo error.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 9]
```diff
@@ -0,0 +1,197 @@
...
+Apuntamos a resolver los problemas relacionados a la transmisión de las transacciones de Monero a través una infraestructura descentralizada de código abierto (redes mesh, satélites, etc.)
+_¿Dónde empezar?_ [taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/108)
+
+### _Únete a la conversación e Involucrate_
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:117]: _Únete a la conversación e **Involúcrate**_

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 10]
```diff
@@ -0,0 +1,197 @@
...
+
+### _Otras maneras de ayudar_
+---
+- Ejecuta un nodo completo - [monerooutreach.org/monero_best_practices/your_own_node.html](https://www.monerooutreach.org/monero_best_practices/your_own_node.html) y [monero.how/how-to-run-monero-node](https://www.monero.how/how-to-run-monero-node)
+- Dona al CCS (Sistema de Financiación Colectiva de la Comunidad) - [ccs.getmonero.org](https://ccs.getmonero.org/)
+- Minar Monero - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Únete a algún equipo de trabajo - [Ir a la sección](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
+- Contribuir a futuras versiones del libro Mastering Monero (Dominando a Monero) - [github.com/monerobook/monerobook](https://github.com/monerobook/monerobook) y [masteringmonero.com](https://masteringmonero.com/)
+- Responder preguntas en Stack Exchange - [monero.stackexchange.com](https://monero.stackexchange.com/)
+- Conviertete en un Mercaderista de guerrilla - [monerooutreach.org/guerrilla-toolkit.html](https://www.monerooutreach.org/guerrilla-toolkit.html)
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:137]: To follow verbs:

_Contribuye_.
_Responde_.

And this is just missing:

_Conviértete_

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 11]
```diff
@@ -0,0 +1,197 @@
...
+- Laboratorio de Investigación de Monero - [web.getmonero.org/es/resources/research-lab](https://web.getmonero.org/es/resources/research-lab/)
+
+Los desarrolladores también podrían estar interesados en participar en alguno de los equipos de trabajo existentes dentro de la comunidad de Monero. Pueden encontrar una lista completa de los más activos más abajo.
+
+### _Equipos de Trabajo_
+
+Abajo encontrarás la lista de los equipo de trabajo de Monero que se actualizaran a medida que nuevos equipos se establezcan. Si tienes una idea de un proyecto para apoyar a Monero que no este en la lista, toma la iniciativa y empieza. Existen algunos recursos disponibles para la comunidad de Monero para ayudar al desarrollo efectivo de los equipos de trabajo y sus operaciones. Si tienen preguntas vayan alguno de los foros y pregunten lo que necesiten. Tenemos decenas de colaboradores que pueden ayudarlos a arrancar.
+
+#### Desarroladores
+Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros software cruciales.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:39]: Grammatical structure suggests, since _software_ is now a Spanish word (anglicism):

_Repositorio de desarrollo de Monero, incluidos el daemon, la CLI, GUI y otros *softwares** cruciales._

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 12]
```diff
@@ -0,0 +1,197 @@
...
+#### Sitio Web
+Desarrollo del sitio web “oficial” (getmonero.org) y su mantenimiento.
+_¿Dónde empezar?_ [repo.getmonero.org/monero-project/monero-site](https://repo.getmonero.org/monero-project/monero-site) y #monero-site
+
+#### Hardware
+Realizando un diseño abierto en especificaciones para un monedero hardware compatible con Monero y otros proyectos relacionados.
+_¿Dónde empezar?_ [taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation]](https://taiga.getmonero.org/project/michael-rfc-hwallet-1-implementation/)
+
+#### Soporte
+Ayudando a los usuarios por una variedad de razones, incuye la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:71]: +Ayudando a los usuarios por una variedad de razones, **incluye** la configuración de monederos, la conexión a nodos, restauración de cuentas, transferencia de fondos y más.+

File: monero-outreach-docs/translations/es/archive/getting_started_es.md [hunk 13]
```diff
@@ -0,0 +1,197 @@
...
+#### GUI
+Mejorando la interfaz gráfica del usuario para el software de Monero.
+_¿Dónde empezar?_ [github.com/monero-project/monero-gui](https://github.com/monero-project/monero-gui), #monero-gui y #monero-dev
+
+#### Proceso de respuesta a vulnerabilidades
+Manejo del proceso de respuesta a vulnerabilidades. Se debe aceptar invitación de parte del equipo de trabajo.
+_¿Dónde comenzar?_ [github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md](https://github.com/monero-project/meta/blob/master/VULNERABILITY_RESPONSE_PROCESS.md)
+
+#### RIAT - Instituto para la Economía futura
+Promoviendo proyectos de cryptomonedas, incluyendo a Monero. Instituto basado en Vienna.
```
Comment (BlackLotus64) [inline monero-outreach-docs/translations/es/archive/getting_started_es.md:90]: +Promoviendo proyectos de **criptomonedas**, incluyendo a Monero. Instituto basado en Vienna._

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:8]: Changed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:19]: Changed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 3 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:21]: Yeah, it also works. It's okay, I'll change it anyway. Changed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 4 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:25]: You're totally right. Changed. Thanks

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 5 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:36]: Suggestions changed. :)

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 7 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:110]: Yep, changed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 6 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:47]: Yeah, I was trying not to translate in singular but while I was reading it again it reads weird when translated to plural instead of singular. I'm making the changes. Thanks.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 8 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:113]: Oups, thanks xD

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 9 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:117]: Fixed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 10 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:137]: Fixed.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 11 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:39]: Fixed, I was confused while trrying to add an 's' and the end of the softwares word. Thank you for clarifying this to me.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 12 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:71]: Thanks.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 13 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md:90]: You're right on this.

**Comment by lh1008** [comment]
Hey @BlackLotus64 

Added the changes and fixed the grammatical mistakes. Take a look at them and make the approval so we merge them :). Thanks man.

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 11 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md]: Yes, I know, it was a little weird, but, well, technical side of the language 😃

File: monero-outreach-docs/translations/es/archive/getting_started_es.md (hunk 6 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/es/archive/getting_started_es.md]: It's common when translating, but you'll also need to be on the other side of the translation, thanks lh1008.

**Comment by lh1008** [comment]
Oh @BlackLotus64 just noticed about the commit of the _rpc_pay_es.md_. I will have to fix it once I push the other translation. Thanks for noticing it xP.
